stopfmt.py

```diff
@@ -7,32 +7,31 @@
 def find_short_ifs(view, settings):
     folds = []
     cur_pt = 0
-    for i in range(1000): # limited in case bugs, could be while True
-        if_stmt = view.find(r" +if[ \(](.*)[ \)]\{\n[^\{\}\n]*\n\s*\}", cur_pt)
+
+    for _ in range(1000): # limited in case bugs, could be while True
+        if_stmt = view.find(r"\s*if[ \(](.*)[ \)]\{\n[^\{\}\n]*\n\s*\}", cur_pt)
```

Comment:
regex needs to `\s` and not match space character. Tabs exist :(

Reply:
also it was missing some cases, you may be better than me at beat saber but I will always beat you at writing regexs!